### PR TITLE
DOC: Remove **kwargs for Stock methods with no params

### PR DESCRIPTION
**Pull request by jackmoody11**
- [ ] closes #xxxx
- [ ] tests added / passed
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [x] added entry to docs/source/whatsnew/vLATEST.txt

### CODE REVIEW

File: docs/source/whatsnew/v0.4.2.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Bug Fixes
```
Comment (addisonlynch) [inline docs/source/whatsnew/v0.4.2.txt]: No need for the entry here...this is not actually a bug (no unexpected/incorrect behavior). Can leave this empty for now.

**Comment by addisonlynch** [comment]
@jackmoody11 also, can you update the docs version to ``0.4.2-dev`` in ``conf.py``? Only update the full version (``release``) such that when the auto-deploy completes for docs it updates the versions when merged. Sorry, forgot to handle this earlier

File: docs/source/whatsnew/v0.4.2.txt (hunk 1 above)
Comment (jackmoody11) [reply docs/source/whatsnew/v0.4.2.txt]: Should I delete this file for now or keep it?

**Comment by jackmoody11** [comment]
> @jackmoody11 also, can you update the docs version to `0.4.2-dev` in `conf.py`? Only update the full version (`release`) such that when the auto-deploy completes for docs it updates the versions when merged. Sorry, forgot to handle this earlier

All done.

**Pull request merged**